Source/WebSocket.swift

```diff
@@ -355,6 +355,9 @@ open class WebSocket : NSObject, StreamDelegate {
                         return
                     }
                 }
+            } else {
```

Comment:
hmmm, I'm not sure how to easily keep the error message. The error would probably make more sense if we could send that error from a stream event that said SSL failed to connect because of the missing ciphers, but that seems like it would have to come from the system :confusing.